bokehjs/src/coffee/renderer/annotation/label.coffee

```diff
@@ -0,0 +1,112 @@
+_ = require "underscore"
+HasParent = require "../../common/has_parent"
+PlotWidget = require "../../common/plot_widget"
+properties = require "../../common/properties"
+
+class LabelView extends PlotWidget
+  initialize: (options) ->
+    super(options)
+    @text_props = new properties.Text({obj: @model, prefix: 'label_'})
+    @fill_props = new properties.Fill({obj: @model, prefix: 'background_'})
+    @line_props = new properties.Line({obj: @model, prefix: 'border_'})
+    @angle_props = new properties.Angle({obj: @model, attr: 'angle'})
```

Comment:
There's some gorpy junk about JS side "properties" that I hope to clear up as part of the new `Document` work in the near future. Short story: line props, etc create several properties at once, and the prefix gives you, e.g., `border_fill_color` but `angle is just a single angle property, called "angle".